app-doc/kicad-doc/kicad-doc-4.0.6.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Electronic Schematic and PCB design tools manuals"
+HOMEPAGE="http://www.kicad-pcb.org/"
+SRC_URI="https://github.com/KiCad/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( GPL-3+ CC-BY-3.0 ) GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="html +pdf"
+LANGS="en fr it ja nl pl"
+for lang in ${LANGS}; do
+	LUSE+=" l10n_${lang}"
+done
+IUSE+=${LUSE}
+REQUIRED_USE="|| ( html pdf ) ^^ ( ${LUSE} )"
+unset lang
+unset LUSE
+
+DEPEND=">=app-text/asciidoc-8.6.9
+	>=app-text/dblatex-0.3.10
+	app-text/texlive:=[l10n_en?,l10n_fr?,l10n_it?,l10n_ja?,l10n_nl?,l10n_pl?]
```

Comment:
yes, definitely

app-text/texlive is for users to pull texlive with their useflags

ebuilds should depend on the dev-texlive/pkg they need